lib/errors.d.ts

```diff
@@ -0,0 +1,21 @@
+declare class AuthenticationFailedError extends Error {
+  message: string;
+}
+declare class BadMessageError extends Error {
+  message: string;
+}
+declare class NoMethodHandlerError extends Error {
+  message: string;
+}
+declare class ResponseError extends Error {
+  res: {};
```

Comment:
I would prefer `any` over `{}` as this implies there is an empty object which I don't believe would be the case.